addons/google_recaptcha/__manifest__.py

```diff
@@ -16,6 +16,11 @@
         'web.assets_frontend': [
             'google_recaptcha/static/src/scss/recaptcha.scss',
             'google_recaptcha/static/src/js/recaptcha.js',
+            # TODO remove next line in master, it was kept here as part of a
+            # stable fix in stable but is unused in the current codebase.
+            'google_recaptcha/static/src/xml/recaptcha.xml',
+        ],
+        'website.assets_wysiwyg': [
```

Comment:
@Guillaume-gdi I doubt `google_recaptcha` depends on `website`...

Reply:
Indeed, I had not thought of that :/ . It should be good now :+1: